### ISSUE TITLE
Heartbeat Not At 500mS, 4 and 8 times Period.

### ISSUE THREAD

**Issue by ForrestErickson** [PubInv/NASA-COG#388]
In the Heartbeat function is working, (toggling an LED) but irregular.
INIT_HEARTBEAT_PERIOD_MS is set for 500mS
Here is the heartbeat toggling the LED_BUILTIN at the fastest rate it can.

![image](https://github.com/PubInv/NASA-COG/assets/5836181/d5eabd99-6a99-499c-81e7-a0252a96a042)
This is the voltage on the D13 LED_BUILTIN.

**Comment by RobertLRead** [comment]
I don't believe this related to SSR duty cycle issue. It remains a potentially dangerous bug.
